enumflags_derive/src/lib.rs

```diff
@@ -137,16 +137,13 @@ fn gen_enumflags(ident: &Ident, item: &DeriveInput, data: &DataEnum) -> TokenStr
             data = wrong_flag_values
         )
     );
-    #[cfg(not(feature = "nostd"))]
-    let std_path = Ident::new("std", span);
-    #[cfg(feature = "nostd")]
-    let std_path = Ident::new("core", span);
+    let std_path = quote_spanned!(span=> self::core);
```

Comment:
I don't think this variable is a good idea anymore. In my opinion it would be cleaner to replace it's occurrences with `self::core` directly.

Reply:
On the other hand, it makes the changes in #5 simpler, which themselves justify the existence of this variable.